instance/models/utils.py

```diff
@@ -52,6 +55,25 @@ def format_help_text(help_text):
     return " ".join(help_text.replace('\n', ' ').split())
 
 
+def _get_setting(name):
+    """
+    Return the Django setting with the given name.
+
+    This function is a helper for default_setting.  It is necessary because Django can't serialize
+    `Settings` objects.
+    """
+    return getattr(settings, name)
+
+
+def default_setting(name):
+    """
+    Return a callable that returns the Django setting with the given name.
+
+    This is useful for default values in model fields.
+    """
+    return functools.partial(_get_setting, name)
```

Comment:
Within the body of this function, what do you think about adding
```python
assert hasattr(settings, name)
```
? 

This seems like a nice opportunity to add a check at app init time that all the referenced settings strings are valid.